### PR TITLE
Update readme and added required vpc_id env var

### PR DESCRIPTION
**Pull request by plentifullee**
*Issue #, if available:*
VPC_ID is required when running ./deploy.sh

*Description of changes:*
Fix messy readme, added bullet point for exporting VPC_ID environment variable.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: walkthroughs/howto-k8s-http2/README.md [hunk 1]
```diff
@@ -6,40 +6,48 @@ This example shows how to manage HTTP/2 routes in App Mesh using Kubernetes depl
...
 
-1. Clone this repository and navigate to the walkthrough/howto-k8s-http2 folder, all commands will be ran from this location
+Clone this repository and navigate to the walkthrough/howto-k8s-http2 folder, all commands will be ran from this location
 1. **Your** account id:
     ```
     export AWS_ACCOUNT_ID=<your_account_id>
     ```
-1. **Region** e.g. us-west-2
+   
+2. **Region** e.g. us-west-2
```
Comment (Y0Username) [inline walkthroughs/howto-k8s-http2/README.md:25]: These numbering format with `1.` were intentional. Markdown knows how to handle it and the benefit is  when we add new items in the middle of the list we'd not want to touch other items.

File: walkthroughs/howto-k8s-http2/deploy.sh [hunk 2]
```diff
@@ -12,6 +12,11 @@ if [ -z $AWS_DEFAULT_REGION ]; then
     exit 1
 fi
 
+if [ -z $VPC_ID ]; then
```
Comment (Y0Username) [inline walkthroughs/howto-k8s-http2/deploy.sh:15]: This being checked further down in the [file](https://github.com/aws/aws-app-mesh-examples/blob/master/walkthroughs/howto-k8s-http2/deploy.sh#L96)

**Comment by Y0Username** [comment]
@plentifullee thanks for the PR, I think this is no-ops unless you have any other concerns.

File: walkthroughs/howto-k8s-http2/README.md (hunk 1 above)
Comment (plentifullee) [reply walkthroughs/howto-k8s-http2/README.md:25]: Thanks I didn't know that! Very useful indeed.

File: walkthroughs/howto-k8s-http2/deploy.sh (hunk 2 above)
Comment (plentifullee) [reply walkthroughs/howto-k8s-http2/deploy.sh:15]: Yes but wouldn't it be better to check beforehand if the variable exist before trying to deploy cloudmap?

**Comment by plentifullee** [comment]
@Y0Username , you are welcome. I'm just trying to see if I can contribute just a little.

File: walkthroughs/howto-k8s-http2/deploy.sh (hunk 2 above)
Comment (Y0Username) [reply walkthroughs/howto-k8s-http2/deploy.sh:15]: Well, `VPC_ID` is only required if the cloudmap namespace doesn't exist. It probably wouldn't make sense to make it required even when that is not required.

**Pull request closed without merging**